linode/manager
# M3-5904: Improve graph accessibility

## Description

**What does this PR do?**
This improves screen reader accessibility for graphs used throughout Cloud Manager. It accomplishes this by making these changes:

* Graphs can be focused via keyboard navigation
* Graphs now have the `graphics-document` ARIA role. This is still technically a [WAI proposal](https://www.w3.org/TR/graphics-aria-1.0/), but has wide support among browsers and assistive tech.
    * `eslint-plugin-jsx-a11y` doesn't yet recognize this role and related roles, but it seems like support will be added in an upcoming release
* Each graph has an ARIA label describing its purpose

For now, the actual contents of the graphs are still inaccessible to screen readers and other assistive tech -- we have another ticket to improve that aspect later.

## How to test

**What are the steps to reproduce the issue or verify the changes?**
* Open Cloud Manager and go to a page that has graphs
    * If you don't have any resources on your account with graphs, the easiest thing to do is to create a Linode and wait a few minutes for the graphs to show up
* Confirm that graphs continue to behave as expected
* Confirm that graphs can be focused via keyboard navigation with tab
* [Enable VoiceOver](https://support.apple.com/guide/voiceover/turn-voiceover-on-or-off-vo2682/mac), confirm that graphs are announced as expected when navigating the page
